docs/getting_started/build_dashboard.md

```diff
@@ -343,36 +352,28 @@ targets:
         kind: scatter
         color: species
         responsive: true
-        height: 450
+        height: 400
       - type: hvplot
         kind: hist
         y: bill_length_mm
         responsive: true
-        height: 350
-      - type: hvplot
-        kind: hist
-        y: bill_depth_mm
-        responsive: true
-        height: 350
-      - type: hvplot
-        kind: hist
-        y: body_mass_g
-        responsive: true
-        height: 350
+        height: 300
+      - type: table
+        show_index: false
+        height: 300
```

Comment:
Can we also update the image?

```suggestion
        height: 300
        theme: midnight
```

Reply:
Just noting that this will also need an update to the `:emphasize-lines`

Reply:
done